src/main/java/com/google/sps/data/User.java

```diff
@@ -2,15 +2,38 @@
 
 import java.util.Collection;
 import java.util.HashSet;
+package profileType;
+
 
 public final class User {
 
-  private final String name;
-  private final Collection<String> productArea;
+public enum profileType{
+    STUDENT("student"),
+    MENTOR("mentor");
+
+    private String profileInput;
+
+    profileType(String input) {
+        this.profileInput = input;
```

Comment:
profileInput seems to be the profileType as a string, yes? Do you intend on using this string value elsewhere? If not, you could probably just have 'public enum type { STUDENT, MENTOR };'

Reply:
Ok. Done